Fabulous.xcodeproj/project.pbxproj

```diff
@@ -391,6 +417,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.duet.fabulous.Fabulous;
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = NO;
```

Comment:
We should probably leave Catalyst support in unless we know specifically it doesn't work

Reply:
Agreed. Was thinking more of appkit when i disabled mac